server/routerlicious/packages/lambdas/src/scribe/checkpointManager.ts

```diff
@@ -10,7 +10,9 @@ import {
     IDocument,
     IScribe,
     ISequencedOperationMessage,
+    runWithRetry,
 } from "@fluidframework/server-services-core";
+import * as winston from "winston";
```

Comment:
I think there's something called `context.log` which is an `ILogger | undefined` that should be used instead of adding winston as a logger here. I think you will need to pass it into Checkpoint manager as a constructor param in LambdaFactory. We can't have winston as an isomorphic dependency